package/kernel/linux/modules/usb.mk

```diff
@@ -72,7 +72,7 @@ $(eval $(call KernelPackage,usb-phy-nop))
 
 define KernelPackage/usb-phy-qcom-dwc3
   TITLE:=DWC3 USB QCOM PHY driver
-  DEPENDS:=@(TARGET_ipq40xx||TARGET_ipq806x) +kmod-usb-dwc3-of-simple
+  DEPENDS:=@(TARGET_ipq40xx||TARGET_ipq806x)
```

Comment:
Don't you need to add `+LINUX_4_14:kmod-usb-dwc3-of-simple` here to not break 4.14 (which you still leave as default)?

Reply:
They are independent and dwc3-of-simple is selected anyway by the soc makefile.

Reply:
Here are builds of this PR:

* https://gitlab.com/ynezz/openwrt/-/jobs/371753867 (with 4.14)
* https://gitlab.com/ynezz/openwrt/-/jobs/371876989 (with 4.19)

Reply:
Now please take the Image Builder (you did build an Image Builder, right?) from the 4.19 build and try to build an image for any device.